.changeset/pink-and-gold.md

```diff
@@ -0,0 +1,5 @@
+---
+'simple-git': minor
+---
+
+Adds vulnerability detection to prevent use of `--upload-pack` and `--receive-pack` without explicitly opting in. 
```

Comment:
Hi there, the `--exec` option should also be blocked https://git-scm.com/docs/git-push#Documentation/git-push.txt---execltgit-receive-packgt